### DISCUSSION TITLE
How to use version 12.7 of postgres

### DISCUSSION THREAD

**Discussion by qmachado** [zalando/postgres-operator#1539] [Q&A]
Hi,

How to use version 12.7 of postgres with operator?
When I set version 12, it uses version 12.4, if I set it to 12.7, the following message appears:

`The postgresql "acid-test-12" is invalid: spec.postgresql.version: Unsupported value: "12.7": supported values: "9.3", "9.4", "9.5", "9.6", "10", "11 ", "12", "13"`

Thanks...

**Comment by MartinX3** [comment]
Currently 12 deploys 12.8, but 12.11 is the newest version.
12.8 is from somewhere last year.
